### DISCUSSION TITLE
Absolute scroll behaves differently from scrolling with the scroll bar in song select

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#18455] [Q&A]
I'm pretty sure this has been the case for a very long time, but is it intentional that the carousel doesn't keep a consistent shape when scrolling quickly with right-click? It seems a little wonky

https://user-images.githubusercontent.com/36758269/170817898-c1ba0f4b-0c2c-4a1d-801c-557a50d3988c.mp4

**Comment by LittleEndu** [comment]
This is seemingly intended behaviour. Right click scrolling is animated. Scrollbar scrolling is not animated.

https://github.com/ppy/osu/blob/62266c72eab593476b1c4039bb717d714632fc88/osu.Game/Graphics/Containers/OsuScrollContainer.cs#L49-L50

https://github.com/ppy/osu-framework/blob/38d1b7fee694b3063b7a3e12a1a7974df1d686d5/osu.Framework/Graphics/Containers/ScrollContainer.cs#L389

Despite the method in `OsuScrollContainer` having a name `scrollFromMouseEvent` it seems it's only used for right click scrolling. If right click scrolling should look the same as scrollbar scrolling, it's probably the best line to change.
